### DISCUSSION TITLE
[A-L] (2023/24) Foglio 5 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#170] [Esercizi - Bertini]
![Screenshot 2023-10-30 171326](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/38ef3163-621c-48d7-937c-7fdc7c022952)

**Comment by daromacarlo** [comment]
# Punto (2) Analogo alla soluzione del prof
![Immagine 2023-11-02 172218](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/5f1fd753-d9dc-4e56-b907-78047705645a)

**Comment by CiottoloMaggico** [comment]
@CarloDaRomadev i punti uno e due dovrebbero essere analoghi, sul terzo ho dei dubbi dato che non ho ben capito il tuo ragionamento.
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/af87da29-6b1c-4ac5-b61f-9be26d925c06)

**Comment by daromacarlo** [comment]
@CiottoloMaggico devo ricontrollare un attimo

**Comment by daromacarlo** [comment]
@CiottoloMaggico il mio ragionamento era: uso la stessa formula usata sopra, in quella sopra si "muove" k con n fissato, in quella sotto si "muove" m con k fissato.

**Comment by Elia-Belli** [comment]
Chiamiamo $\mathbb{P}(V)=p_v=1/2$ la probabilità che un cittadino si rechi a votare e $\mathbb{P}(SI|V)=p_s=1/2$ la probabilità che voti si andando a votare.
- **(1)** $\mathbb{P}(V \cap SI)=\mathbb{P}(V)\mathbb{P}(SI|V)=1/2\cdot 1/2=1/4$
- **(2)** Innanzitutto per avere $k$ SI è necessario che almeno $k$ individui su $n$ si rechino a votare: realizziamo quindi uno schema di Bernoulli sull'evento binario _"va a votare oppure no"_ con $p_v=1/2$ e selezioniamo solo i casi in cui si recano più di k persone.
$$\mathbb{P}(almeno\ k\ votanti)=\sum^n_{i=k}\binom{n}{i}p_v^i(1-p_v)^{n-i}=\sum^n_{i=k}\binom{n}{i}p_v^n$$
Tra gli $k\leq m \leq n$ che si sono presentati adesso vogliamo che esattamente $k$ di loro votino SI: realizziamo un'altro schema di Bernoulli sull'evento binario _"vota si oppure no"_ con $p_s=1/2$ e selezioniamo solo i casi in cui otteniamo esattamente k SI.
$$\mathbb{P}(k\ SI\ su\ m\ votanti )=\binom{m}{k}p_s^k(1-p_s)^{m-k}= \binom{m}{k}p_s^m$$
A questo punto fissato $k$ calcoliamo la probabilità di ottenere $k$ SI al variare dei votanti $m$:
$$\mathbb{P}(k\ SI)=\sum^n_{m=k}\binom{n}{m}p_v^n\cdot \binom{m}{k}p_s^m$$
Quindi quello che facciamo ad ogni iterazione della sommatoria è selezionare $m$ cittadini e costringerli a votare e poi tra questi ne selezioniamo $k$ e li costringiamo a votare si, ciò per ogni possibile numero di votanti. (Dubbio: l'evento di ottenere k si da m votanti è disgiunto dall'ottenere k si da m+1 votanti?)
> Inserendo dei valori in quest'ultima formula ottengo la stessa probabilità data dalla formula di @CarloDaRomadev

**Comment by Elia-Belli** [comment]
# Soluzione alternativa del prof al punto (3)
Il numero di votanti è $m=k+h$ con $k=\\#SI$ e $h=\\#NO$ 
I potenziali elettori per il No sono $n-k$, questi hanno votato No o si sono astenuti.
Se chiamiamo $p=\mathbb{P}(singolo\ elettore\ vota\ NO|non\ ha\ votato\ SI)=\frac{1/4}{1/2+1/4}=1/3$:
$$\mathbb{P}(\\#votanti=m|\\#SI=k)=\binom{n-k}{h}p^h(1-p)^{n-k-h}$$
